### ISSUE TITLE
Add ability to override default path to postgresql install

### ISSUE THREAD

**Issue by connorblack** [pgxman/pgxman#225]
https://github.com/pgxman/pgxman/blob/5f197fcb9deb17d416c3c7def7a42c36a15b4b26/internal/pg/version.go#L26C23-L26C42

Hi - I'm currently unable to use pgx after switching over to a different postgresql image (bitnami) that installs postgres in `/opt/bitnami/postgresql` instead of `/usr/lib/postgresql`. It would be great to have a flag you can pass to override the path.

**Comment by owenthereal** [comment]
👍 I think this is a good thing to add. cc: @wuputah

**Comment by wuputah** [comment]
I dont think this is easily possible in the near futures since pgxman is currently using apt packages, and those packages all have hardcoded paths, because they're also based on Postgres packages that has hardcoded paths, etc. 

So I think this feature request is more like "add bitnami support". I don't know anything about bitnami so I can't guess what would be involved to do that, but I would think it's a lot more involved than just this one function for where pg_config is located.

**Comment by connorblack** [comment]
@wuputah @owenthereal I think it might be as simple as that. Bitnami is just a collection of container images maintained by the VMWare team that have pretty minimal changes from the upstream images. 

This specific image is debian as well, haven't had any issue apt installing pg packages directly in the container: https://hub.docker.com/r/bitnami/postgresql
